code/API_definitions/qod-provisioning.yaml

```
@@ -57,6 +57,16 @@ info:
     ### Restrictions for tokens without an associated authenticated identifier:
 
     - For scenarios which do not have a single device identifier associated to the token during the authentication flow, e.g. 2-legged access tokens, the `device` object MUST be provided in the API request. This ensures that the device identification is explicit and valid for each API call made with these tokens.
+
+    # Multi-SIM scenario handling
+
+    In multi-SIM scenarios where more than one mobile device is associated with a phone number (e.g. a smartphone with an associated smartwatch), it might not be possible to uniquely identify the device to which the QoS provisioning should apply from that phone number. If the phone number is used as the device identifier when provisioning in a multi-SIM scenario, the API may respond with an error, or may apply the provisioning to a device other than the intended device.
+
+    Possible workarounds in such a scenario include:
+    - Using the authorisation code flow to obtain an access token, which will automatically identify the intended device

```

If we finally approve in Commonalities that a device identifier in request body is incompatible with a 3-legged access token (and "In this case, a `422 UNNECESSARY_IDENTIFIER` error code MUST be returned"), then we can remark that 
```suggestion
    - Using the authorisation code flow to obtain an access token, which will automatically identify the intended device. Moreover, in this case, no other device identifier must be included in the request.
```

---

The proposed changes are a patch to the Fall 24 Meta-Release version, where a device identifier is allowed even if a 3-legged token is used. Rather than repeat the rule here, the proposal is to rely on the dedicated section in the `info.description` to explain it.